app/controllers/uploaded_files_controller.rb

```diff
@@ -0,0 +1,69 @@
+class UploadedFilesController < ApplicationController
+  before_action :set_uploaded_file, only: [:show, :edit, :update, :destroy]
+
+  # GET /uploaded_files
+  def index
+    @uploaded_files = UploadedFile.all
+  end
+
+
+  # GET /uploaded_files/1
+  def show
+  end
+
+
+  # GET /uploaded_files/new
+  def new
+    @uploaded_file = UploadedFile.new
+  end
+
+
+  # GET /uploaded_files/1/edit
+  def edit
+  end
+
+
+  # POST /uploaded_files
+  def create
+    @uploaded_file = UploadedFile.new(uploaded_file_params)
+
+    if @uploaded_file.save
+      redirect_to @uploaded_file, notice: 'Uploaded file was successfully created.'
+    else
+      render :new
+    end
+  end
+
+
+  # PATCH/PUT /uploaded_files/1
+  def update
+    if @uploaded_file.update(uploaded_file_params)
+      redirect_to @uploaded_file, notice: 'Uploaded file was successfully updated.'
+    else
+      render :edit
+    end
+
+  end
+
+
+  # DELETE /uploaded_files/1
+  def destroy
+    @uploaded_file.destroy
+    redirect_to uploaded_files_url, notice: 'Uploaded file was successfully destroyed.'
+  end
+
+
+  private
+  # Use callbacks to share common setup or constraints between actions.
+  def set_uploaded_file
+    @uploaded_file = UploadedFile.find(params[:id])
+  end
+
+
+  # Never trust parameters from the scary internet, only allow the white list through.
+  def uploaded_file_params
+    params.require(:uploaded_file).permit(:title, :description, :actual_file, :membership_application)
+  end
+
+
+end
```

Comment:
No, of course not.  But that guideline doesn't exist in isolation.  That assumes that the user story is more rigorous than is the reality of this project.  IMO, there has not been the time or process or conversations necessary to develop the business goals and user stories.  (I don't know if that's because this is in part a learning project, or if that's a deliberate decision or a combination of those two.)

Conventions over configuration isn't just about saving time.  It also speaks to the the fact that the scaffold code is there because most projects have found it necessarily and valid.  And using code that is mature and well vetted reduces risk and saves time. 

In this particular project, I think the costs and risks of including code produced by the very standard rails scaffolding are very, very low. It is a small enough set of code that it doesn't confuse or impact readability.  It is functionality that most projects need (even if not at their very first iteration). If some particular performance testing points to serious gains by removing the code, then it should absolutely be culled.

I still totally support revising the generators and templates to whatever is appropriate for this project.  Otherwise this will simply be an ongoing issue for code reviews, as developers who come in to this project are going to use standard rails tools, which includes the generators.  To me that may be the best thing to do:  This is the 2nd signal that there's a problem in our process, so let's find and fix the root of the problem.